src/test/java/data/provider/CrocodileProvider.java

```diff
@@ -0,0 +1,14 @@
+package data.provider;
+
+import com.github.javafaker.Faker;
+import data.models.CreateCrocodileRequest;
+
+public class CrocodileProvider {
+    public static CreateCrocodileRequest prepareCrocodileRequest() {
+        CreateCrocodileRequest createCrocodileRequest = new CreateCrocodileRequest();
+        createCrocodileRequest.setName(Faker.instance().funnyName().name());
+        createCrocodileRequest.setSex("M");
+        createCrocodileRequest.setDateOfBirth("2020-12-12");
```

Comment:
Koristi i ovde nesto za random, faker ili LocalDate klasu da resis to, ako se ne snadjes cimni da pomognem